### ISSUE TITLE
Support of multiple conditions for term aggregation order

### ISSUE THREAD

**Issue by cjost1988** [ruflin/Elastica#1478]
Hello,

I figured out that elasticsearch is allowing multiple conditions for term aggregation orders.
But as far as I understand `Aggregation\Terms::setOrder()` is not capable of this feature.

Are there any plans to support this? 

Cheers,
Christian

**Comment by ruflin** [comment]
You are right that there is not a named API function for this today but you can already make use of it by using `setParam` on your Terms object. All setOrder does is using setParam and setting one array:

https://github.com/ruflin/Elastica/blob/9e840a1b9bc051baef16473c66a0084fc87cf091/lib/Elastica/Aggregation/Terms.php#L21

On your end you can pass an array with your values:

```
$terms->setParam('order', [$order1 => $direction1, $order2 => $direction2]);
```

I haven't tested it but that should work.

Interested to contribute a `setOrders` or `addOrder` method?

**Comment by cjost1988** [comment]
@ruflin Thank you for your quick response.

Yes, I already forked your repo and added these methods on my own :) I didn't wanted to call the `setParam`/`addParam` methods directly to keep the interface clean.

I'll add tests and will provide a PR when it's ready from my side.

**Comment by cjost1988** [comment]
I'll add an additional method `setOrders` beside `setOrder` which takes an array of order definitions to not add a BC => could be merged without adding BC release.

This also guarantees no broken queries if a user will execute both `setOrder` and `setOrders` at the same time. Just overwriting his previous definition.
An `addOrder` beside the current `setOrder` method could potentially lead to wrong queries.

**Comment by ruflin** [comment]
SGTM. Looking forward to the PR.

Let's skip `addOrder` until we see a use case for it.

**Comment by cjost1988** [comment]
This got resolved in: [#1480](https://github.com/ruflin/Elastica/pull/1480)
